src/season/season.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Get } from '@nestjs/common';
+import { SeasonService } from './season.service';
+
+@Controller('season')
+export class SeasonController {
+  constructor( private seasonService: SeasonService) {}
+
+  @Get()
+  async getSeasons() {
```

Comment:
We need the league ID, right ?

Reply:
No, This API just retrieves all seasons (3 seasons) with their league id